.travis.yml

```diff
@@ -7,4 +7,4 @@ install:
   - pip install flake8
 
 script:
-  - flake8 --max-line-length=125 ./
+  - flake8 --ignore=E226,E302,E41 --max-line-length=120 --exclude=tests/* --max-complexity=15 ./
```

Comment:
А зачем ты игнорируешь эти правила? И зачем тебе вообще `max-complexity`?

Reply:
В самом конце второго домашнего задания (после пункта 7) был этот пример настроек. Подумала, что, раз оно есть в задании, то надо добавить и посмотреть, что будет

E226 - это missing whitespace around arithmetic operator
E302 - это expected 2 blank lines, found 0
E41 - это all E41* codes will be ignored, but since currently there are no E41* codes, that's a noop

Reply:
Так, это я не удалил их, извини, просто пример копировал, нужный кусок из него вынул, а остатки забыл удалить

Reply:
Ага, поняла, тогда в настройках оставила только длину в 125